debian/control

```diff
@@ -0,0 +1,29 @@
+Source: nyan
+Priority: optional
+Maintainer: Matthias Geiger <matthias.geiger1024@tutanota.de>
+Build-Depends:	debhelper-compat (= 13),
+		cmake (>= 3.18),
+		flex (>=2.6)
+Standards-Version: 4.6.0
+Section: libs
+Homepage: https://github.com/SFTtech/nyan
+#Vcs-Browser: https://salsa.debian.org/debian/nyan
+#Vcs-Git: https://salsa.debian.org/debian/nyan.git
```

Comment:
do we also want to list the github-remote here, or just salsa? i don't know the conventions